schema/content-descriptor.json

```diff
@@ -6,14 +6,15 @@
   "properties": {
     "mediaType": {
       "description": "the mediatype of the referenced object",
-      "$ref": "defs-image.json#definitions/mediaType"
+      "$ref": "defs-image.json#/definitions/mediaType"
     },
     "size": {
       "description": "the size in bytes of the referenced object",
       "$ref": "defs.json#/definitions/int64"
     },
     "digest": {
-      "$ref": "defs-image.json#definitions/digest"
+      "description": "the cryptographic checksum digest of the object, in the pattern '<hash>:<hexadecimal digest>'",
```

Comment:
I carry on the description from https://github.com/opencontainers/image-spec/blob/master/schema/defs-image.json#L10 directly. I think it had better to keep same words for one entry.